vulture_os/toolkit/api_parser/waf_cloudflare/waf_cloudflare.py

```diff
@@ -79,7 +77,13 @@ def get_logs(self, logs_from=None, logs_to=None, test=False):
         if logs_to:
             query.update({'end': logs_to.strftime('%Y-%m-%dT%H:%M:%SZ')})
 
-        query.update({"fields":"BotScore,BotScoreSrc,BotTags,CacheCacheStatus,CacheResponseBytes,CacheResponseStatus,CacheTieredFill,ClientASN,ClientCountry,ClientDeviceType,ClientIP,ClientIPClass,ClientMTLSAuthCertFingerprint,ClientMTLSAuthStatus,ClientRequestBytes,ClientRequestHost,ClientRequestMethod,ClientRequestPath,ClientRequestProtocol,ClientRequestReferer,ClientRequestScheme,ClientRequestSource,ClientRequestURI,ClientRequestUserAgent,ClientSSLCipher,ClientSSLProtocol,ClientSrcPort,ClientTCPRTTMs,ClientSrcPort,ClientXRequestedWith,Cookies,EdgeCFConnectingO2O,EdgeColoCode,EdgeColoID,EdgeEndTimestamp,EdgePathingOp,EdgePathingSrc,EdgePathingStatus,EdgeRateLimitAction,EdgeRateLimitID,EdgeRequestHost,EdgeResponseBodyBytes,EdgeResponseBytes,EdgeResponseCompressionRatio,EdgeResponseContentType,EdgeResponseStatus,EdgeServerIP,EdgeStartTimestamp,EdgeTimeToFirstByteMs,FirewallMatchesActions,FirewallMatchesRuleIDs,FirewallMatchesSources,OriginIP,JA3Hash,OriginDNSResponseTimeMs,OriginRequestHeaderSendDurationMs,OriginIP,OriginResponseBytes,OriginResponseDurationMs,OriginResponseHTTPExpires,OriginResponseHTTPLastModified,OriginResponseHeaderReceiveDurationMs,OriginResponseStatus,OriginResponseTime,OriginSSLProtocol,OriginTCPHandshakeDurationMs,OriginTLSHandshakeDurationMs,ParentRayID,RayID,RequestHeaders,ResponseHeaders,SecurityLevel,UpperTierColoID,SmartRouteColoID,WAFAction,WAFFlags,WAFMatchedVar,WAFRuleID,WAFProfile,WAFRuleMessage,WorkerCPUTime,WorkerStatus,WorkerSubrequest,WorkerSubrequestCount,ZoneID,ZoneName"})
+        r = self.session.get(f"https://api.cloudflare.com/client/v4/zones/{self.waf_cloudflare_zoneid}/logs/received/fields")
+        if r.status_code != 200:
+            logger.error(f"{[__parser__]}:get_logs: Status code = {r.status_code}, Error = {r.text}",
+                         extra={'frontend': str(self.frontend)})
+            r.raise_for_status()
+
+        query['fields'] = r.json().keys()
```

Comment:
Please prefer creating and using a new `get_available_fields()` function instead of putting this in the `get_logs()` function